components/authz-service/server/v2/policy.go

```diff
@@ -110,7 +110,12 @@ func NewPoliciesServer(
 	case storage.SuccessfulBeta1:
 		v = api.Version{Major: api.Version_V2, Minor: api.Version_V1}
 	case storage.Successful:
-		v = api.Version{Major: api.Version_V2, Minor: api.Version_V0}
+		// auto-upgrade 2.0 installs to 2.1
```

Comment:
Language ambiguity makes the phrase challenging to parse; consider:
```suggestion
		// auto-upgrade a 2.0 installation to 2.1
```